<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QDialog" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_main" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>220</height>
               </size>
              </property>
              <property name="title">
               <string>串口配置</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>20</y>
                 <width>131</width>
                 <height>141</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QComboBox" name="port_comboBox"/>
                </item>
                <item>
                 <widget class="QComboBox" name="buadrate_comboBox"/>
                </item>
                <item>
                 <widget class="QComboBox" name="databits_comboBox"/>
                </item>
                <item>
                 <widget class="QComboBox" name="parity_comboBox"/>
                </item>
                <item>
                 <widget class="QComboBox" name="stopbits_comboBox"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>50</width>
                 <height>141</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>端口：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>波特率：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>数据位：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>校验位：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>停止位：</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>160</y>
                 <width>181</width>
                 <height>51</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QPushButton" name="open_port_Button">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="check_port_Button">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>检测</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>调试信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>16</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
               <item row="0" column="1" colspan="2">
                <widget class="QPlainTextEdit" name="plainTextEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>二维码显示</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="rencode_view">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="rencode_view_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Image</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="rencode_lineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="recode_lineEdit_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="print_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>打印</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
